### PR TITLE
new map "jungle-botany"

### PR DESCRIPTION
**Pull request by Lloserhub**
## About The Pull Request

This PR would add a new jungle ruin into the game

## Why It's Good For The Game

It adds a some variety for the ruins and this one also has a few custom thing on the map, such as crampled notes.

## Changelog
:cl:
add: New jungle ruin centered around botany
/:cl:

### CODE REVIEW

**Comment by Ms-Mee** [comment]
You changed the headers, you should've written below them not over them
Only the seeds and special fertilizers are rare, the rest is obtainable via autolathe and a cigarette vendor
Remove the template changelog bits and make it the following
add: New jungle ruin centered around botany
Or something similar

**Comment by Lloserhub** [comment]
> You changed the headers, you should've written below them not over them Only the seeds and special fertilizers are rare, the rest is obtainable via autolathe and a cigarette vendor Remove the template changelog bits and make it the following add: New jungle ruin centered around botany Or something similar

Okay, so I updated the description and messed around with the GitHub by accident. But I did what you asked for!

File: shiptest.dme [hunk 1]
```diff
@@ -14,6 +14,7 @@ g// DM Environment file for shiptest.dme.
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\RandomRuins\JungleRuins\jungle_botany.dmm"
```
Comment (ZephyrTFA) [inline shiptest.dme]: Dont tick your map file.
Instead add a ruin datum for your ruin.

Comment (Lloserhub) [reply shiptest.dme]: Got it, let me change that

Comment (Lloserhub) [reply shiptest.dme:17]: Where do I add the datum?

**Pull request merged**